.evergreen/install-dependencies.sh

```diff
@@ -90,9 +93,6 @@ else
   mv "${NODE_ARTIFACTS_PATH}/${node_directory}" "${NODE_ARTIFACTS_PATH}/nodejs"
 fi
 
-export PATH="$NODE_ARTIFACTS_PATH/npm_global/bin:$NODE_ARTIFACTS_PATH/nodejs/bin:$PATH"
-hash -r
```

Comment:
Just noting this is because this path update is already done in `init-node-and-npm-env`